apps/frontend/.env.sample

```
@@ -18,3 +18,4 @@ VITE_DEFAULT_DOCUMENT_PROCESSING_MODEL=...
 VITE_FEATURE_FLAG_MCP_PARLA_ALLOWED=...
 VITE_FEATURE_FLAG_SPLASH_SCREEN_ALLOWED=...
 VITE_SPLASH_CONTENT_URL=...
+VITE_SPLASH_API_COMMIT_URL=...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SPLASH_API_COMMIT_URL key should go before the VITE_SPLASH_CONTENT_URL key

(UnorderedKey)